docs/index.md

```diff
@@ -6,7 +6,7 @@ This is a data collection app that uses your organization's web maps and the Arc
 
 ### Generic application
 
-The app was designed to work in a generic context and thus your organization can configure the app to consume your own web map, out of the box. To accomplish this, the web map is configured by a set of rules that the app adheres to, driving the app's behavior. These rules are defined by the map's definition and by the map's layers' popup configurations. To learn more about what drives the app's behavior, read the section entitled [_Using Map Definition & Popup Configurations to Drive App Behavior_](#using-map-definition-popup-configurations-to-drive-app-behavior).
+The app was designed to work in a generic context and thus your organization can configure the app to consume your own web map, out of the box. To accomplish this, the web map is configured by a set of rules that the app adheres to, driving the app's behavior. These rules are defined by the map's definition and by the map's layers' pop-up configurations. To learn more about what drives the app's behavior, read the section entitled [_Using Map Definition & Pop-up Configurations to Drive App Behavior_](#using-map-definition-popup-configurations-to-drive-app-behavior).
```

Comment:
I think this in-document linking text needs to have the `popup` also hyphenated for this to work. I just tried on my local machine and it worked if it's cited as:

```diff
- #using-map-definition-popup-configurations-to-drive-app-behavior
+ #using-map-definition--pop-up-configurations-to-drive-app-behavior
```

I always refer to this gist's comment to remind myself of the markdown rules for generating in-document hyperlinks: https://gist.github.com/asabaylus/3071099#gistcomment-1593627